waspc/src/Wasp/Generator/Valid/PackageJson.hs

```diff
@@ -3,172 +3,187 @@ module Wasp.Generator.Valid.PackageJson
   )
 where
 
-import Control.Applicative ((<|>))
-import Data.List (intercalate)
+import Control.Selective (whenS)
 import qualified Data.Map as M
 import qualified Data.Set as S
+import qualified Validation as V
 import qualified Wasp.ExternalConfig.Npm.PackageJson as P
 import Wasp.Generator.DepVersions (prismaVersion, typescriptVersion)
 import Wasp.Generator.Monad (GeneratorError (GenericGeneratorError))
 import qualified Wasp.Generator.NpmWorkspaces as NW
 import Wasp.Generator.ServerGenerator.DepVersions (expressTypesVersion)
-import Wasp.Generator.WebAppGenerator.DepVersions (reactRouterVersion, reactTypesVersion, reactVersion, viteVersion)
+import Wasp.Generator.Valid.Validator
```

Comment:
One more thought here, let's look at the line somewhat below:
```hs
getValidationErrors validateFile pkgJson
```

One thing I find hard is to at glance understand which functions are local here, and which are external. I would maybe prefer if it was:
```hs
V.getValidationErrors validateFile pkgJson
```

So qualifying everything with `V`.
If often do taht on purpose in Haskell, so when reading, I know what is local and what not.

Sometimes it doesn't make sense if library is well known, or we use it everywhere, and so on, but this one is not super known so it can help with readibility.

I am nto sure all the members of the team would agree with me though, so I will leave this to you, this is just a suggestion, do as you like.

RAW

Reply:
did it